src/lib/objects/payggy/standalone.svelte

```diff
@@ -18,43 +18,53 @@
 	import { formatAddress, formatTokenAmount } from '$lib/utils/format'
 	import type { WakuObjectArgs } from '..'
 	import {
-		SendTransactionStandaloneSchema,
-		type SendTransactionStandalone,
 		type MessageDataSend,
+		SendTransactionStoreSchema,
+		type SendTransactionStore,
 	} from './schemas'
 	import type { Token } from '../schemas'
+	import { defaultBlockchainNetwork } from '$lib/adapters/transaction'
+	import { throwError } from '$lib/utils/error'
 
-	export let args: WakuObjectArgs<MessageDataSend, MessageDataSend>
+	export let args: WakuObjectArgs<SendTransactionStore, MessageDataSend>
 
-	let store: SendTransactionStandalone
+	let store: SendTransactionStore | undefined
 	$: {
-		const res = SendTransactionStandaloneSchema.safeParse(args.store)
-		if (res.success) {
-			store = res.data
-		} else {
-			console.error(res.error)
+		if (args.store) {
+			const res = SendTransactionStoreSchema.safeParse(args.store)
+			if (res.success) {
+				store = res.data
+			} else {
+				console.error(res.error)
+			}
 		}
 	}
 
 	let token: Token
 	let amount = ''
 	let fee: Token | undefined = undefined
-	$: if (!token) token = store.nativeToken
-	$: if (args.estimateTransaction && amount && token) {
+	$: if (!token) {
+		token =
+			args.tokens.find((token) => token.symbol === defaultBlockchainNetwork.nativeToken) ||
+			throwError('invalid token')
+	}
+	let toUser =
+		args.users.find((user) => user.address !== args.profile.address) || throwError('invalid user')
+	$: if (amount && token) {
 		const tokenToTransfer = { ...token, amount: BigInt(Number(amount) * 10 ** token.decimals) }
-		args.estimateTransaction(store.toUser.address, tokenToTransfer).then((f) => (fee = f))
+		args.estimateTransaction(toUser.address, tokenToTransfer).then((f) => (fee = f))
 	}
-	$: if (!amount && store.view === 'overview') history.back()
+	// $: if (!amount && store && store.view === 'overview') history.back()
```

Comment:
This is here in case user refreshes which makes the `amount` an empty string. The assumption was, that user should be navigated to the page, where they fill the amount in.

Reply:
I can't replicate that issue. But I did notice that the URL change is wrong now on main. It goes from `/chat/koKPzjnB57rInHObAJMY/object/payggy/new` to `/chat/koKPzjnB57rInHObAJMY/object/new/new/overview`

Reply:
I added a comment to the issue #178 explaining the problem. As I write there it would be better to come up with a generic solution to this, because it is very easy to introduce a change that currently breaks the page when reloading.

I also opened a new issue for the URL change problem: #179